Jint/Runtime/Interpreter/Statements/JintIfStatement.cs

```diff
@@ -5,19 +5,19 @@ namespace Jint.Runtime.Interpreter.Statements
 {
     internal sealed class JintIfStatement : JintStatement<IfStatement>
     {
-        private JintStatement _statementConsequent = null!;
+        private ProbablyBlockStatement _statementConsequent;
         private JintExpression _test = null!;
-        private JintStatement? _alternate;
+        private ProbablyBlockStatement? _alternate;
 
         public JintIfStatement(IfStatement statement) : base(statement)
         {
         }
 
         protected override void Initialize(EvaluationContext context)
         {
-            _statementConsequent = Build(_statement.Consequent);
+            _statementConsequent = new ProbablyBlockStatement(_statement.Consequent);
```

Comment:
Do you really need to initialize everything upfront, even if it is not needed?

Reply:
Initialize happens lazily, in this case not sure about perf benefit to delay more.